testsuite/test-validate.sh

```diff
@@ -558,6 +558,19 @@ __check_exports_hbase () {
     fi
 }
 
+__check_exports_hive () {
```

Comment:
Could this test check be more significant?   Like is there a text that ensures the benchmark ran?  Like "read 1000 rows" or something like that?